src/ScanCommand.php

```diff
@@ -49,6 +49,26 @@ protected function configure()
                 InputOption::VALUE_OPTIONAL,
                 'The maximum number of seconds the request can take',
                 10
+            )
+            ->addOption(
+                'user-agent',
+                'u',
+                InputOption::VALUE_OPTIONAL,
+                'The User Agent to pass for the request',
+                ''
+            )
+            ->addOption(
+                'skip-verification',
```

Comment:
Let's rename this `verify-ssl` with a default of `true`

Reply:
Can't have a default true for flags right?

Reply:
Oh, you're right... 😄